specification/datamigration/resource-manager/Microsoft.DataMigration/preview/2018-07-15-preview/definitions/Services.json

```diff
@@ -157,10 +157,15 @@
         "virtualSubnetId": {
           "type": "string",
           "description": "The ID of the Microsoft.Network/virtualNetworks/subnets resource to which the service should be joined"
+        },
+        "virtualNicId": {
+          "type": "string",
+          "description": "The ID of the Microsoft.Network/networkInterfaces resource to which the service have"
         }
       },
       "required": [
-        "virtualSubnetId"
+        "virtualSubnetId",
+        "virtualNicId"
```

Comment:
This should not be required property. It is not needed as part of input.

Reply:
Since this should not be 'required', this will no longer be a breaking change.

Reply:
Thanks for pointing out, will change it